pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/participant/ParticipantWrapper.java

```diff
@@ -140,43 +140,46 @@ private void addWrapperToList(UnparsedESParticipantDto elasticSearchParticipantD
         result.add(participantWrapperDto);
     }
 
-    private void addWrapperToList(ElasticSearchParticipantDto elasticSearchParticipantDto, List<ParticipantWrapperDto> result, DDPInstanceDto ddpInstanceDto) {
+    private void addWrapperToList(ElasticSearchParticipantDto elasticSearchParticipantDto, List<ParticipantWrapperDto> result, DDPInstanceDto ddpInstanceDto,
+                                  boolean hideMedicalRecordRequestTracking) {
 
         elasticSearchParticipantDto.getDsm().ifPresent(esDsm -> {
             Participant participant = esDsm.getParticipant().orElse(new Participant());
 
-            esDsm.getOncHistory().ifPresent(oncHistory -> {
-                participant.setCreated(oncHistory.getCreated());
-                participant.setReviewed(oncHistory.getReviewed());
-            });
+            List<ParticipantData> participantData = esDsm.getParticipantData();
+            sortBySelfElseById(participantData);
 
             StudyPreFilter.fromPayload(StudyPreFilterPayload.of(elasticSearchParticipantDto, ddpInstanceDto))
                     .ifPresent(StudyPreFilter::filter);
 
-            List<MedicalRecord> medicalRecord = esDsm.getMedicalRecord();
-            List<OncHistoryDetail> oncHistoryDetails = esDsm.getOncHistoryDetail();
             List<KitRequestShipping> kitRequestShipping = esDsm.getKitRequestShipping();
-            List<Tissue> tissues = esDsm.getTissue();
-            List<SmId> smIds = esDsm.getSmId();
-            List<ClinicalOrder> clinicalOrder = esDsm.getClinicalOrder();
-
-            mapSmIdsToProperTissue(tissues, smIds);
-
-            mapTissueToProperOncHistoryDetail(oncHistoryDetails, tissues);
 
-
-
-            List<ParticipantData> participantData = esDsm.getParticipantData();
-            sortBySelfElseById(participantData);
             ParticipantWrapperDto participantWrapperDto = new ParticipantWrapperDto();
             participantWrapperDto.setEsData(elasticSearchParticipantDto);
             participantWrapperDto.setParticipant(participant);
-            participantWrapperDto.setMedicalRecords(medicalRecord);
-            participantWrapperDto.setOncHistoryDetails(oncHistoryDetails);
-            participantWrapperDto.setKits(kitRequestShipping);
             participantWrapperDto.setParticipantData(participantData);
-            participantWrapperDto.setAbstractionActivities(Collections.emptyList());
-            participantWrapperDto.setAbstractionSummary(Collections.emptyList());
+
+            if (hideMedicalRecordRequestTracking) {
```

Comment:
Shouldn't this be `if (!hideMedicalRecordRequestTracking) {`??

Reply:
Good catch. changed name to make it easier readable